ibc-core/ics02-client/types/src/events.rs

```diff
@@ -100,6 +165,35 @@ impl From<ConsensusHeightAttribute> for abci::EventAttribute {
     }
 }
 
+impl TryFrom<abci::EventAttribute> for ConsensusHeightAttribute {
+    type Error = ClientError;
```

Comment:
Just a nit to add a newline between the associated type and the function signature.
```suggestion
    type Error = ClientError;
    
```